invenio_github/config.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
```

Comment:
I did not spend a lot of time looking at these configs. 
I think some can be removed and others are not being picked up (e.g. `GITHUB_REMOTE_APP["signup_handler"]["setup"]`)